src/main/java/mekanism/common/content/transporter/TItemStackFilter.java

```diff
@@ -20,18 +20,10 @@
 
     @Override
     public boolean canFilter(ItemStack itemStack, boolean strict) {
-        if (itemStack.isEmpty()) {
-            return false;
-        }
-
-        if (strict && sizeMode) {
-            if (max == 0 || itemStack.getCount() < min) {
-                return false;
-            }
-        }
-
-        return (itemType.getHasSubtypes() ? itemType.isItemEqual(itemStack)
-              : itemType.getItem() == itemStack.getItem());
+        return super.canFilter(itemStack, strict) &&
+                !(strict && sizeMode && (max == 0 || itemStack.getCount() < min)) &&
+                (itemType.getHasSubtypes() ?
```

Comment:
This subtype check chain (from `itemType.getHasSubtypes() `) should be replaced with `net.minecraftforge.items.ItemHandlerHelper#canItemStacksStackRelaxed`